terratag.go

```diff
@@ -211,34 +223,40 @@ func tagFileResources(path string, args *common.TaggingArgs) (*counters, error)
 		if err := file.ReplaceWithTerratagFile(path, text, args.Rename); err != nil {
 			return nil, err
 		}
+
 		perFileCounters.taggedFiles = 1
 	} else {
 		log.Print("[INFO] No taggable resources found in file ", path, " - skipping")
 	}
+
 	return &perFileCounters, nil
 }
 
 func toHclMap(tags string) (string, error) {
 	var tagsMap map[string]string
-	err := json.Unmarshal([]byte(tags), &tagsMap)
-	if err != nil {
+
+	if err := json.Unmarshal([]byte(tags), &tagsMap); err != nil {
 		// If it's not a JSON it might be "key1=value1,key2=value2".
 		tagsMap = make(map[string]string)
 		pairs := strings.Split(tags, ",")
+
 		for _, pair := range pairs {
 			match := pairRegex.FindStringSubmatch(pair)
 			if match == nil {
 				return "", fmt.Errorf("invalid input tags! must be a valid JSON or pairs of key=value.\nInput: %s", tags)
 			}
+
 			tagsMap[match[1]] = match[2]
 		}
 	}
 
 	keys := utils.SortObjectKeys(tagsMap)
 
-	var mapContent []string
+	mapContent := []string{}
```

Comment:
How come you don't need the `var` here?

Reply:
ahh good question!
`:=` is shortcut that does not need var.